spec/system/admin/products_v3/update_spec.rb

```diff
@@ -526,8 +526,10 @@
           # Client side validation
           click_button "Save changes"
           within new_variant_row do
+            # In CI we get "Please fill out this field." and locally we get
+            # "Please fill in this field."
```

Comment:
😕  That's weird.
In the test environment, we use locale `en_TEST` which inherits everything from default local `en`.
In dev, we use `en_AU` (unless you have specified otherwise). I couldn't find any reason for this in `en_AU.yml`.

Hmm, but wait, when you run the tests locally it's still the same RAILS_ENV=TEST.

I give up, this fix looks good 👍

Reply:
🤦 Ohhh, this is a browser message, not Rails. 
The browser message is dependent on the system locale. This will likely cause problems for devs in other countries, like Filipe and Ahmed. 

So I think we can't assert the message contents. But the JS API should at least provide a way to see if it's valid or not.

Reply:
`±` I think we should fix this as it will cause confusion for new contributors.

Reply:
Fair enough, I ended removing the check for the message, I just check there is one present.